vendor/comit_i/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+authors = ["Franck Royer <franck@coblox.tech>"]
```

Comment:
Rude.

Reply:
Do `cargo init` locally and you'll see.

Reply:
Of course you'd want to claim comit-i yours.

Reply:
God no!